packages/prisma/zod/eventtype.ts

```diff
@@ -30,6 +30,8 @@ export const _EventTypeModel = z.object({
   requiresConfirmation: z.boolean(),
   disableGuests: z.boolean(),
   minimumBookingNotice: z.number().int(),
+  beforeEventBuffer: z.number().int(),
```

Comment:
this supposed to be commited here?

Reply:
Probably not no - whoever last merged didn't run prisma generate and this got picked up

Reply:
@zomars is it ok if this is contributed here?

Reply:
These files are supposed to be gitignored cc @emrysal

Reply:
https://github.com/calcom/cal.com/blob/a311f6bf4b10e03bcfd449e4785c1ab1bb653b0a/.gitignore#L67